pkg/kubelet/kubelet_test.go

```diff
@@ -228,18 +226,15 @@ func newTestKubeletWithImageList(
 	}
 	// setup eviction manager
 	evictionManager, evictionAdmitHandler, err := eviction.NewManager(kubelet.resourceAnalyzer, eviction.Config{}, killPodNow(kubelet.podWorkers), kubelet.imageManager, fakeRecorder, nodeRef, kubelet.clock)
-	if err != nil {
-		t.Fatalf("failed to initialize eviction manager: %v", err)
-	}
+	assert.NoError(t, err, "Failed to initialize eviction manager")
```

Comment:
A fatal error is the equivalent of the `require` checks. Is it worth continuing the test if this fails?